src/main/java/seedu/address/model/ReadOnlyTrackIter.java

```diff
@@ -20,4 +21,12 @@
      * This list will not contain any duplicate modules.
      */
     ObservableList<Module> getModuleList();
+
+    /**
+     * Returns an unmodifiable view of the tasks list.
+     * This list will not contain any duplicate tasks.
+     *
+     * @return
+     */
+    ObservableList<Task> getTaskList();
```

Comment:
Can delete the `@return` here as well, or add the return value

Reply:
It will be fixed in the next PR!